server/src/grpc/meta_event_service/mod.rs

```diff
@@ -249,21 +253,44 @@ async fn handle_open_shard(ctx: HandlerContext, request: OpenShardRequest) -> Re
             shard_id: shard_info.id,
             cluster_version: topology.cluster_topology_version,
         };
-        schema
-            .open_table(open_request.clone(), opts.clone())
-            .await
-            .box_err()
-            .with_context(|| ErrWithCause {
-                code: StatusCode::Internal,
-                msg: format!("fail to open table, open_request:{open_request:?}"),
-            })?
-            .with_context(|| ErrNoCause {
-                code: StatusCode::Internal,
-                msg: format!("no table is opened, open_request:{open_request:?}"),
-            })?;
+        let result = schema.open_table(open_request.clone(), opts.clone()).await;
+
+        match result {
+            Ok(Some(_)) => {
+                success += 1;
+            }
+            Ok(None) => {
+                fail += 1;
+                error!("no table is opened, open_request:{open_request:?}");
+                err_list.push(table.name);
+            }
+            Err(e) => {
+                fail += 1;
+                error!("fail to open table, open_request:{open_request:?}, err:{e}");
+                err_list.push(table.name);
+            }
+        };
     }
 
-    Ok(())
+    info!(
+        "Open shard finish, shard id:{}, cost:{}ms, successful tables:{}, failed tables:{}",
+        shard_info.id,
+        instant.saturating_elapsed().as_millis(),
+        success,
+        fail
+    );
+
+    if err_list.is_empty() {
+        Ok(())
+    } else {
+        Err(Error::OpenShardErr {
+            code: StatusCode::Internal,
+            msg: format!(
+                "Failed to open shard:{},  because of failed tables:{err_list:?}",
+                shard_info.id
```

Comment:
```suggestion
                "Failed to open shard:{shard_info.id}, because of failed tables:{err_list:?}",
```

Reply:
This will result in error msg: `invalid format string: field access isn't supported`.